### PR TITLE
IGVF-1449-large-scale-reference

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/construct_library_set.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`construct_library_set.json`*
 
+### Minor changes since schema version 4
+
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/construct_library_set.md:5]: Add ReferenceFile linkTo to `large_scale_gene_list` and `large_scale_loci_list`.

File: src/igvfd/schemas/changelogs/mixins.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## Changelog for *`mixins.json`*
 
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`. (02/14/2024)
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/mixins.md:3]: Add ReferenceFile linkTo to large_scale_gene_list and large_scale_loci_list.

File: src/igvfd/schemas/changelogs/prediction_set.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`prediction_set.json`*
 
+### Minor changes since schema version 4
+
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/prediction_set.md:5]: Add ReferenceFile linkTo to large_scale_gene_list and large_scale_loci_list.

**Review by gabdank — CHANGES_REQUESTED** [review]
monir change to changelog

File: src/igvfd/schemas/changelogs/prediction_set.md (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/prediction_set.md]: changed

File: src/igvfd/schemas/changelogs/mixins.md (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/mixins.md]: changed

File: src/igvfd/schemas/changelogs/construct_library_set.md (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/construct_library_set.md]: changed

**Review by gabdank — APPROVED** [review]

**Pull request merged**